src/scaladoc/scala/tools/nsc/doc/html/resource/lib/template.css

```diff
@@ -666,7 +666,7 @@ div#definition > h4#signature > span.modifier_kind > i.unfold-arrow,
 }
 
 .cmt code {
-  font-weight: bold;
+  font-family: "Source Code Pro", "Monaco", "Ubuntu Mono Regular", "Lucida Console", monospace;
```

Comment:
This is is the same as for `<pre>` (https://github.com/scala/scala/blob/2.13.x/src/scaladoc/scala/tools/nsc/doc/html/resource/lib/template.css#L616)